### DISCUSSION TITLE
我需要帮助，“检测到v2ray-core”的版本并非v5，请使用v5版本或将v2ray降级至v1.5“

### DISCUSSION THREAD

**Discussion by Niezhanyang** [v2rayA/v2rayA#953] [Q&A]
我需要帮助
问题如下：

“检测到v2ray-core”的版本并非v5，请使用v5版本或将v2ray降级至v1.5“
v2rayA服务端正在运行，Version:2.0.5

请问谁知道怎么降级或者使用v5版本的v2ray-core

我这边使用的是raspberry pi os 32bit。

我的v2Ray 版本
V2Ray 4.28.2 (V2Fly, a community-driven edition of V2Ray.) Custom (go1.15.2 linux/arm)
A unified platform for anti-censorship.

输出日志
`023/05/19 12:38:50.917  [I] [pre.go:160]  init DB
2023/05/19 12:38:52.110  [A] [main.go:26]  V2Ray binary is /usr/local/bin/v2ray
2023/05/19 12:38:52.111  [A] [main.go:26]  V2Ray asset directory is /run/user/0/v2raya
2023/05/19 12:38:52.111  [A] [main.go:26]  v2rayA working directory is /
2023/05/19 12:38:52.111  [A] [main.go:26]  v2rayA configuration directory is /etc/v2raya
2023/05/19 12:38:52.111  [A] [main.go:26]  Golang: go1.20.2
2023/05/19 12:38:52.111  [A] [main.go:26]  OS: linux
2023/05/19 12:38:52.111  [A] [main.go:26]  Arch: arm
2023/05/19 12:38:52.111  [A] [main.go:26]  Lite: false
2023/05/19 12:38:52.111  [A] [main.go:26]  Version: 2.0.5
2023/05/19 12:38:52.111  [A] [main.go:26]  Starting...
2023/05/19 12:38:52.112  [I] [main.go:27]  the core was not running the last time v2rayA exited
2023/05/19 12:38:52.217  [A] [index.go:116]  v2rayA is listening at http://127.0.0.1:2017
2023/05/19 12:38:52.218  [A] [index.go:116]  v2rayA is listening at http://192.168.50.17:2017
2023/05/19 12:38:52.218  [A] [index.go:116]  v2rayA is listening at http://192.168.50.204:2017
2023/05/19 12:38:52.218  [A] [index.go:116]  v2rayA is listening at http://[::1]:2017
2023/05/19 12:38:52.218  [A] [index.go:116]  v2rayA is listening at http://[fe80::d4cb:177:b179:a8be]:2017
2023/05/19 12:38:52.218  [A] [index.go:116]  v2rayA is listening at http://[fe80::eafe:404a:6f89:cf2b]:2017
2023/05/19 12:39:42.740  [E] [account.go:79]  length of password should be between 6 and 32
2023/05/19 12:40:59.714  [W] [context.go:165]  PostConnection: failed to connect: core version too low: the version 4.28.2 is>
2023/05/19 12:40:59.714  [E] [connection.go:36]  failed to connect: failed to connect: core version too low: the version 4.28>
2023/05/19 12:41:06.470  [W] [context.go:165]  PostConnection: failed to connect: core version too low: the version 4.28.2 is>
2023/05/19 12:41:06.471  [E] [connection.go:36]  failed to connect: failed to connect: core version too low: the version 4.28>
2023/05/19 12:41:08.523  [W] [context.go:165]  PostConnection: failed to connect: core version too low: the version 4.28.2 is>
2023/05/19 12:41:08.524  [E] [connection.go:36]  failed to connect: failed to connect: core version too low: the version 4.28>
2023/05/19 12:41:13.828  [W] [context.go:165]  PostConnection: failed to connect: core version too low: the version 4.28.2 is>
2023/05/19 12:41:13.828  [E] [connection.go:36]  failed to connect: failed to connect: core version too low: the version 4.28>

`

**Comment by Niezhanyang** [comment]
对于这个问题我已经解决了，参考[https://github.com/v2rayA/v2rayA/issues/794](url)。
由于网络问题我只能手动安装
先下载这个安装脚本，[https://github.com/v2fly/fhs-install-v2ray/blob/master/install-release.sh](url)
然后去[https://github.com/v2fly/v2ray-core/releases](url)，下载对应的包我需要的是[[v2ray-linux-arm32-v7a.zip](https://github.com/v2fly/v2ray-core/releases/download/v5.4.1/v2ray-linux-arm32-v7a.zip)](url)

然后参考[https://dsencheng.github.io/15936838298753.html](url)

先卸载
sudo bash install-release.sh --remove
sudo  apt purge curl unzip

再手动安装
sudo bash install-release.sh --local v2ray-linux-arm32-v7a.zip
启动
sudo systemctl start v2ray
sudo systemctl status v2ray


虽然没有报错然而好像并没有什么用，使用我的订阅链接我并没有翻出去
我很肯定我的梯子可以使用，他在windows和ubuntu下运行的很好，有没有大佬知道怎没解决

还没有尝试降回1.5版本之后再测试一下。

**Comment by banneryyyyy** [comment]
如果你是ubuntu的话 可以直接使用sudo apt-get remove v2ray先移除旧版本  然后使用sudo apt-get install v2ray=5.10.1下载回来 当然这个5.10.1是使用apt-cache madison先看v5现在哪一个可以下载的 根据实际情况来

**Reply by Red-T007** [reply]
apt-cache madison什么都没有

**Reply by Red-T007** [reply]
https://aisikao.ren/22633/我看了这篇文章，不报错了

**Comment by k3vinPeng** [comment]
这个问题我也遇到了，我解决了。如果你使用debian或者ubuntu的话用apt install v2ray安装的是v2ray/v2ray-core。但是v2raya用的是v2fly/v2ray-core。他们是两个不同的项目。所以你需要手动去v2fly/v2ray-core项目里下载，并手动安装它。就可以了。
